target/linux/ath79/image/generic.mk

```diff
@@ -813,6 +813,16 @@ define Device/engenius_ecb1750
 endef
 TARGET_DEVICES += engenius_ecb1750
 
+define Device/engenius_ecb600
+  $(Device/engenius_loader_okli)
```

Comment:
If you use engenius_loader_okli here, please put the device after the common definition.

Reply:
should the common definition be in front of all engenius board definitions? even ones that don't apply to them?